cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -57,14 +57,28 @@ function deploy_cnao() {
         $kubectl create -f /opt/cnao/operator.yaml
 
         if [ "$KUBVIRT_WITH_CNAO_SKIP_CONFIG" != "true" ]; then
-            $kubectl create -f /opt/cnao/network-addons-config-example.cr.yaml
+            if [ "$KUBEVIRT_WITH_MULTUS_V3" == "true" ]; then
+                # set CNAO to not deploy Multus components to prevent conflicts with deploying Multus from manifests.
+                # FIXME: Remove this workaround when CNAO provides Multus v4 GA version.
+                create_no_multus_cluster_network_addons_cr
+            else
+                $kubectl create -f /opt/cnao/network-addons-config-example.cr.yaml
+            fi
```

Comment:
Let's move the condition inside a function so this is more clear
```suggestion
            compose_network_addons_config | $kubectl apply -f -
```

Reply:
done